coderd/database/migrations/000168_pg_coord_tailnet_v2_api.up.sql

```diff
@@ -0,0 +1,72 @@
+BEGIN;
+
+CREATE TYPE tailnet_status AS ENUM (
+	'ok',
+	'lost'
+);
+
+CREATE TABLE tailnet_peers (
+	id uuid NOT NULL,
+	coordinator_id uuid NOT NULL,
+	updated_at timestamp with time zone NOT NULL,
+	node bytea NOT NULL,
+	status tailnet_status DEFAULT 'ok'::tailnet_status NOT NULL,
+	PRIMARY KEY (id, coordinator_id),
+	FOREIGN KEY (coordinator_id) REFERENCES tailnet_coordinators(id) ON DELETE CASCADE
+);
+
+-- For shutting down / GC a coordinator
+CREATE INDEX idx_tailnet_peers_coordinator ON tailnet_peers (coordinator_id);
+
+-- Any time tailnet_peers table changes, send an update with the affected peer ID.
+CREATE FUNCTION tailnet_notify_peer_change() RETURNS trigger
```

Comment:
Do we have prior usage of functions and triggers in the codebase? Should we be worried about customers having difficulties upgrading due to the new permissions required on the coder PSQL user?

Reply:
This is how the existing PG Coordinator works, so no change in permissions.